### DISCUSSION TITLE
Is WASM debugging supported?

### DISCUSSION THREAD

**Discussion by xperiandri** [unoplatform/uno#6462] [Q&A]
I see that in the samples app https://github.com/unoplatform/uno/blob/master/src/SamplesApp/SamplesApp.Wasm/Properties/launchSettings.json there is a debugger configuration present.
On which versions of Uno and .NET is it supported?

**Comment by jeromelaban** [comment]
It works on .NET 5+ TargetFramework with bootstrapper 3.0-dev builds.

**Reply by xperiandri** [reply]
Cool! I will try
